views/blm.html

```diff
@@ -0,0 +1,42 @@
+<main class="jumbotron homepage PRIMER-REMOVE-ME">
+
+  <div class="py-7 py-md-8 py-lg-9">
+    <div class="container-lg p-responsive">
+      <div class="container-lg p-responsive position-relative">
+        <div class="row">
+          <div class="text-center">
+            <h2 class="h2-mktg text-normal mt-0 mb-4">
+              <a class="text-white" href="https://www.gofundme.com/f/georgefloyd">George Floyd.</a>
+            </h2>
+          </div>
+          <p class="h3-mktg text-normal mt-0 mb-4">
+            Natosha McDade, Yassin Mohamed, Finan H. Berhe, Sean Reed, Steven Demarco Taylor, Breonna Taylor, Ariane McCree,
+            Terrance Franklin, Miles Hall, Darius Tarver, William Green, Samuel David Mallard, Kwame Jones, De’von Bailey,
+            Christopher Whitfield, Anthony Hill, De’Von Bailey, Eric Logan, Jamarion Robinson, Gregory Hill Jr, JaQuavion Slaton,
+            Ryan Twyman, Brandon Webber, Jimmy Atchison, Willie McCoy, Emantic Fitzgerald Bradford J, D’ettrick Griffin, Jemel
+            Roberson, DeAndre Ballard, Botham Shem Jean, Robert Lawrence White, Anthony Lamar Smith, Ramarley Graham, Manuel Loggins
+            Jr, Trayvon Martin, Wendell Allen, Kendrec McDade, Larry Jackson Jr, Jonathan Ferrell, Jordan Baker, Victor White III,
+            Dontre Hamilton, Eric Garner, John Crawford III, Michael Brown, Ezell Ford, Dante Parker, Kajieme Powell, Laquan
+            McDonald, Akai Gurley, Tamir Rice, Rumain Brisbon, Jerame Reid, Charly Keunang, Tony Robinson, Walter Scott, Freddie
+            Gray, Brendon Glenn, Samuel DuBose, Christian Taylor, Jamar Clark, Mario Woods, Quintonio LeGrier, Gregory Gunn, Akiel
+            Denkins, Alton Sterling, Philando Castile, Terrence Sterling, Terence Crutcher, Keith Lamont Scott, Alfred Olango,
+            Jordan Edwards, Stephon Clark, Danny Ray Thomas, DeJuan Guillory, Patrick Harmon, Jonathan Hart, Maurice Granton, Julius
+            Johnson, Jamee Johnson, Michael Dean...
+          </p>
+          <span></span>
+          <p class="h3-mktg text-normal mt-0 mb-4">
+            The Electron community believes that Black Lives Matter. To be silent is to be complicit. 
+            It is our responsibility and our honor to use our platform to amplify the voices of the oppressed and marginalized. 
+          </p>
+          <p class="h3-mktg text-normal mt-0 mb-4">
+            Please donate to <a class="text-white" href="https://blacklivesmatter.com/">Black Lives Matter</a>, 
+            <a class="text-white" href="https://www.naacpldf.org/">the NAACP Legal Defense and Educational
+              Fund</a>, <a class="text-white" href="https://eji.org/">the Equal Justice Initiative</a>,
+              <a class="text-white" href="https://minnesotafreedomfund.org/">the Minnesota Freedom Fund</a>, and
```

Comment:
```suggestion
              , and
```

See https://minnesotafreedomfund.org/donate: They are no longer able to accept donations and would like us to promote other organizations (which we're doing here).

Reply:
Ah thank you! I'll change that

Reply:
Here's a pretty extensive list of ways to donate/help: https://blacklivesmatters.carrd.co/